state/workingset.go

```diff
@@ -45,9 +45,9 @@ type (
 		Version() uint64
 		Height() uint64
 		// General state
-		State(hash.PKHash, State) (State, error)
+		State(hash.PKHash, interface{}) error
+		PutState(hash.PKHash, interface{}) error
 		CachedState(hash.PKHash, State) (State, error)
```

Comment:
can we also refactor to CachedState( , interface{})? 
I know we may deprecate it after using actionHandler, but better to make it same interface for sake of consistency?